src/main/java/heronarts/lx/color/GradientUtils.java

```diff
@@ -193,25 +197,53 @@ public enum BlendMode {
     }),
 
     HSV((c1, c2, lerp) -> {
+      float hue1 = c1.hue;
+      float hue2 = c2.hue;
+      float sat1 = c1.saturation;
+      float sat2 = c2.saturation;
+      float bright1 = c1.brightness;
+      float bright2 = c2.brightness;
+      if (c1.isBlack()) {
+        hue1 = hue2;
+        sat1 = 0;
+        bright1 = 0;
```

Comment:
Isn't this (and bright2 below) always a NOP?

Reply:
Thanks!  Fixed.